spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/store/redis/RedisTokenStore.java

```diff
@@ -30,11 +31,10 @@
 	private static final String AUTH_TO_ACCESS = "auth_to_access:";
 	private static final String AUTH = "auth:";
 	private static final String REFRESH_AUTH = "refresh_auth:";
-	private static final String ACCESS_TO_REFRESH = "access_to_refresh:";
 	private static final String REFRESH = "refresh:";
 	private static final String REFRESH_TO_ACCESS = "refresh_to_access:";
-	private static final String CLIENT_ID_TO_ACCESS = "client_id_to_access:";
-	private static final String UNAME_TO_ACCESS = "uname_to_access:";
+	private static final String CLIENT_ID_TO_ACCESS = "client_id_to_access_z:";
```

Comment:
Changing the Key ID for both `CLIENT_ID_TO_ACCESS` and `UNAME_TO_ACCESS` will leave existing keys orphaned after a user upgrades to this version, which will ultimately break existing applications. Is there a reason why the Key ID's need to be changed?

Reply:
If they're not changed then existing applications will also break, because they'll the wrong data types at those keys.